lms/djangoapps/discussion_api/api.py

```diff
@@ -556,6 +552,12 @@ def _handle_voted_field(form_value, cc_content, api_content, request, context):
     )
 
 
+def _handle_read_field(form_value, user, cc_content):
+    """mark thread as read for the user"""
+    if form_value:
+        user.read(cc_content)
```

Comment:
@jcdyer Thread is only marked as unread when anyone add response/comment, report/Unreport, vote/unvote and thats being handled implicitly. There is no explicit UI to mark thread as unread (as far as current behaviour is concerned).

May be we can add a business feature to allow user to explicitly mark thread back to unread. What do you think?
cc: @robrap @scottrish